site/content/docs/5.3/forms/validation.md

```diff
@@ -90,7 +90,7 @@ Custom feedback styles apply custom colors, borders, focus styles, and backgroun
     <div class="form-check">
       <input class="form-check-input" type="checkbox" value="" id="invalidCheck" required>
       <label class="form-check-label" for="invalidCheck">
-        Agree to terms and conditions
+        <a href="" target="_blank">Agree to terms and conditions</a>
```

Comment:
```suggestion
        <a href="#" target="_blank" aria-label="Terms and conditions (new window)">Agree to terms and conditions</a>
```

Reply:
See: H83: [Using the target attribute to open a new window on user request and indicating this in link text](https://www.w3.org/TR/WCAG20-TECHS/H83.html)

Reply:
I think it's better to have the `href="#"` because that's what we usually do, see for instance https://deploy-preview-2002--boosted.netlify.app/docs/5.3/utilities/link/ or https://deploy-preview-2002--boosted.netlify.app/docs/5.3/helpers/stretched-link/ or https://deploy-preview-2002--boosted.netlify.app/docs/5.3/helpers/icon-link/ or https://deploy-preview-2002--boosted.netlify.app/docs/5.3/helpers/colored-links/ etc.

But you are right, it doesn't open a new window/tab in all these pages. There is probably a `preventDefault()` to add somewhere... 🤔

Reply:
Update: the link will open a new window/tab even if you have just `href=""`  🙃